packages/vscode-extension/src/commonlib/m365Login.ts

```diff
@@ -270,6 +270,21 @@ export class M365Login extends BasicLogin implements M365TokenProvider {
       );
       if (tokenRes.isOk()) {
         const tokenJson = ConvertTokenToJson(tokenRes.value);
+        // if token is empty, try to get token by app studio scope, normally this should only affect UX
```

Comment:
`TODO` here since there should be some follow-up fixes?
Or, create work item to track the follow-ups?

Reply:
I have created two work items to follow this.
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/24223342/
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/24223343/